met-web/src/components/common/index.tsx

```diff
@@ -178,30 +178,90 @@ export const modalStyle = {
     overflowY: 'scroll',
     color: Palette.text.primary,
 };
-
-export const MetHeader1 = styled(Typography)(() => ({
-    fontSize: '2.3rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
-
-export const MetHeader2 = styled(Typography)(() => ({
-    fontSize: '1.9rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
-
-export const MetHeader3 = styled(Typography)(() => ({
-    fontSize: '1.5rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
-
-export const MetHeader4 = styled(Typography)(() => ({
-    fontSize: '1.3rem',
-    fontWeight: 500,
-    fontFamily: "'BCSans', 'Noto Sans', Verdana, Arial, sans-serif",
-}));
+interface HeaderProps {
+    sx?: SxProps;
+    children?: React.ReactNode | string;
+    align?: 'left' | 'right' | 'inherit' | 'center' | 'justify';
+    style?: React.CSSProperties;
+    onClick?: () => void;
+    color?: string;
+}
+export const MetHeader1 = ({ children, sx, onClick, color, align, style }: HeaderProps) => {
+    return (
+        <Typography
+            sx={{
+                ...sx,
+                fontSize: '2.3rem',
+                fontWeight: 500,
```

Comment:
can we define constants for this? I mean for 500

Reply:
done